js/pages/tablespage.ts

```diff
@@ -227,11 +229,10 @@ export class TablesPage extends PageBase implements ICurrentTableProvider {
         app.tabBar.visible(true);
         app.tabBar.select("tables", false);
         app.processing(true);
-        const oldColor = $(".progress-background").css("background-color");
         $(".progress-background").css("background-color", "black");
         timeService.setTimeout(() => {
             app.processing(false);
-            $(".progress-background").css("background-color", oldColor);
+            $(".progress-background").css("background-color", this.progressBackgroundColor());
```

Comment:
Here should be just updating progressBackgroundColor. No jQuery, Knockout should take care of updates

Reply:
So i should change main project then and add style binding for all .progress-background ?

Reply:
Yes